source/help_and_support/__index.md

```diff
@@ -29,10 +29,6 @@ To unsubscribe from a mailing list, send email to `LIST-unsubscribe@jena.apache.
 
 Full details of Apache mailing lists: [http://www.apache.org/foundation/mailinglists.html](http://www.apache.org/foundation/mailinglists.html).
 
-## Other channels
-
-Some Jena developers hang out on `#jena` on `irc.freenode.net`.
-
```

Comment:
BTW, there's a Slack channel I think? Or there was? Maybe I'm confusing with another project, but in case I'm not, should we mention that here (if not already)?

Reply:
This was the "quick fix" to [JENA-2235](https://issues.apache.org/jira/browse/JENA-2235).

There is a slack channel and AFAIK, we can have multiple. We haven't really discussed whether to use it and what for. dev@ discussion?

A "synchronous users@" channel may not be the best for the users, if questions don't get responses or it is seen as an alternative to bug reports with test cases.